### DISCUSSION TITLE
Smart Wall Switch Touch Screen 4 Groups Switch 8-way Scene

### DISCUSSION THREAD

**Discussion by Dimmers81** [Koenkk/zigbee2mqtt#24325] [Devices]
Hey, I wanted to buy this on AliExpress: a 4-gang Zigbee touch monitor switch. I tried searching the Zigbee2MQTT device list, but I know sometimes the images didn't match the device, and I was wondering if it is compatible or will be. Thanks in advance as I could don't find any markings or anything to see if it would work. #FingersCrossed
https://www.aliexpress.com/item/1005007630716149.html

**Comment by Vitalijus66** [comment]
Hello, I would also be interested in connecting such a switch to zigbee2mqtt. Can anyone help solve this problem? How is it possible to do this? Can anyone advise where to look for a solution? In Zigbee2mqtt FE page  this device is represented as Unsupported.
Thanks in advance.
![Capture](https://github.com/user-attachments/assets/d974d3e0-37f9-4263-b015-0395ff08aff6)

**Comment by Westcott1** [comment]
Even better if it's ESP32 with graphics.

**Reply by Vitalijus66** [reply]
do you have any example?
